Gopkg.lock

```diff
@@ -32,6 +32,60 @@
   revision = "1952afaa557dc08e8e0d89eafab110fb501c1a2b"
 
 [[projects]]
+  branch = "master"
```

Comment:
This large block of additions looks a lot like fluxsvc dependencies. Did you run `dep ensure` with some service code still around by accident?